### ISSUE TITLE
Using the Tab key to navigate through the questionnaires, which have somw `math` role, will highlight the mathematical expresions

### ISSUE THREAD

**Issue by jaguarfi** [apluslms/a-plus-rst-tools#108]
If we use the keyboard to navigate through the questionnaires that have some `math` role,  all the math expressions will be recognized as interactive elements. This should not happen because the mathematical expressions are just part of the text.

![keyboard_navigation_math_inside_questionnaires](https://user-images.githubusercontent.com/10437475/101627942-c64cf580-3a27-11eb-97cb-f6dc673ad95e.gif)

**Comment by jaguarfi** [comment]
Moreover, in this particular case, when we use a screen reader, the screen reader do not read out the answers. It only says  "checkbox, no checked"

**Comment by Mankro** [comment]
The MathJax.js library has already been developed with accessibility in mind. It is intentional that tab navigation finds them. I don't know about screen readers, though I thought that MathJax.js should have good support for screen readers. It may depend on the screen reader software and configurations too. Nevertheless, we don't develop MathJax, but A+ could ensure that MathJax uses accessible default configurations. Courses might change the MathJax configurations, but the platform should define accessible defaults.

**Comment by jaguarfi** [comment]
Check this out https://docs.mathjax.org/en/v2.7-latest/misc/accessibility-features.html#mathjax-accessibility-extensions

**Comment by Mankro** [comment]
@jaguarfi Then we want to enable the MathJax accessibility extensions. It looks like they should improve accessibility without any downsides, so let's try it.
